smdebug/tensorflow/keras.py

```diff
@@ -725,6 +727,7 @@ def _remove_fetches_and_callbacks(self, mode):
 
     def _prepare_collections_for_tf2(self):
         self._prepare_collections()
+        self.logger.debug("SMDEBUG: Preparing Collections")
```

Comment:
Case of "smdebug" is inconsistent. There should be some higher level wrapper that manages prepending "smdebug" so that it isn't expected for the writer of the log to remember to prepend "smdebug".